cmd/kubeadm/app/cmd/phases/init/kubeletready.go

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package phases
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	clientcmd "k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog"
+	"k8s.io/kubernetes/cmd/kubeadm/app/cmd/options"
+	"k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow"
+	cmdutil "k8s.io/kubernetes/cmd/kubeadm/app/cmd/util"
+	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
+	kubeletphase "k8s.io/kubernetes/cmd/kubeadm/app/phases/kubelet"
+)
+
+var (
+	kubeletReadyPhaseExample = cmdutil.Examples(`
+		# Updates settings relevant to the kubelet after TLS bootstrap"
+		kubeadm init phase kubelet-ready all --config
+		`)
+)
+
+// NewKubeletReadyPhase creates a kubeadm workflow phase that updates settings
+// relevant to the kubelet after TLS bootstrap.
+func NewKubeletReadyPhase() workflow.Phase {
+	return workflow.Phase{
+		Name:    "kubelet-ready",
```

Comment:
the new phase names are debatable. opinions?

Reply:
no strong opinions, but since we already have `kubelet-[verb]` for `kubelet-start`, and almost all nouns for the existing sub-phases, probably we should follow the same pattern also here. e.g. 
```
kubelet-complete
    certificate-rotation (or cert-rotation)
```

Reply:
Sgtm. Comments by others?

Reply:
kubelet-finalize